components/raftstore-v2/src/worker/pd/misc.rs

```diff
@@ -33,6 +33,8 @@ where
         let causal_ts_provider = self.causal_ts_provider.clone();
         let logger = self.logger.clone();
         let shutdown = self.shutdown.clone();
+        let log_interval = Duration::from_secs(1);
```

Comment:
I think 1 seconds may be still too much. Maybe 5 seconds?

Reply:
ok